Editor/EditorCore/ObjExporter.cs

```diff
@@ -351,10 +351,17 @@ static string WriteMtlContents(IEnumerable<Model> models, ObjOptions options, ou
                     }
                 }
 
-                if (mat.HasProperty("_Color"))
+                Color color = mat.color;
+                if (mat.HasProperty("_BaseColorMap") || mat.HasProperty("_BaseMap")) // HDRP || URP
+                {
+                    color = mat.GetColor("_BaseColor");
```

Comment:
Does `"_BaseMap"` also need to be looked at here?

Reply:
this how it is done in the model exporter code